### ISSUE TITLE
NTR: Liver biopsy: ornithine transcarbamylase activity low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9216]
New term request Liver biopsy: ornithine transcarbamylase activity low (elthat_110106021347): OTC low on hepatic biopsy  UMLS: 1839535

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000334
- label: Reduced hepatic ornithine transcarbamylase activity
- parents: [Abnormal liver enzyme activity or concentration (HP:0034685)](https://hpo.jax.org/app/browse/term/HP:0034685)
- pmid: PMID:6733950
- def: Activity or concentration of ornithine transcarbamylase in the liver below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9216
- synonyms:
